sys/include/net/gnrc/netif.h

```diff
@@ -41,6 +41,11 @@ extern "C" {
 #endif
 
 /**
+ * @brief   Identifier of the loopback interface
+ */
+#define GNRC_NETIF_LOOPBACK_PID     (-1)
```

Comment:
Considering that I chose `-1` as `NETIF_INVALID` in https://github.com/RIOT-OS/RIOT/pull/6413, I would actually prefer `KERNEL_PID_LAST + 1` now.